apps/challenges/views.py

```diff
@@ -42,6 +42,12 @@ def challenge_list(request, challenge_host_team_pk):
         return paginator.get_paginated_response(response_data)
 
     elif request.method == 'POST':
+
+        if not ChallengeHost.objects.filter(user=request.user).filter(team_name=challenge_host_team_pk):
```

Comment:
i think this condition also needs to be updated. chaining filter is not bad, but it is not required here.
you can simply pass both conditions in a single `filter`

also you cannot compare `team_name` with `challenge_host_team_pk` because `team_name` is a challenge host team instance where as `challenge_host_team_pk` is integer. So you can either convert it to `team_name=challenge_host_team` or `team_name_pk=challenge_host_team_pk`. I will prefer the latter one,

one more thing which i want to point out is you can use `exists` to check if there is any row. For more reference, you can see this [link](http://stackoverflow.com/a/6869696/2534102)